content/events/2020/03/2020-03-19-uswds-monthly-call-march-2020.md

```diff
@@ -3,18 +3,18 @@
 # Learn how to edit our pages at https://workflow.digital.gov
 slug: uswds-monthly-call-march-2020
 title: "USWDS Monthly Call - Color Tokens Update"
-deck: "Join U.S. Web Design System Product Lead, Dan Williams, as he talks about the design system and answers your questions."
-summary: "Join U.S. Web Design System Product Lead, Dan Williams, as he talks about the design system and answers your questions."
+deck: "Watch as U.S. Web Design System Product Lead, Dan Williams, as he talks and answers questions about the design system."
```

Comment:
I think "Watch as U.S." should just be "Watch U.S." . Like the `summary`.

Reply:
agreed!